clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/Keys.groovy

```diff
@@ -88,12 +91,12 @@ class Keys {
             break
           case Namespace.SUBNETS.ns:
             if (parts.length == 5) {
-              result << [id: parts[2], account: parts[3], region: parts[4]]
+              result << [account: parts[2], region: parts[3], id: parts[4]]
             }
             break
           case Namespace.NETWORKS.ns:
             if (parts.length == 5) {
-              result << [id: parts[2], account: parts[3], region: parts[4]]
+              result << [account: parts[2], region: parts[3], id: parts[4]]
```

Comment:
What is preferred order of the parts? `account`, `region`, `id` or `id`, `account`, `region`? you are doing both.